tests: Bluetooth: Tester: Format BTP cmd/evt/rsp logs as hex

Thalley opened the pull request:
Since the values are defined as hex, e.g. 0x82, it is easier to compare with the log if they also log them as such.

Thalley committed:
tests: Bluetooth: Tester: Format BTP cmd/evt/rsp logs as hex

Since the values are defined as hex, e.g. 0x82, it is easier
to compare with the log if they also log them as such.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

jhedberg commented on tests/bluetooth/tester/src/btp.c:
```diff
@@ -96,8 +96,8 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 
-		LOG_DBG("cmd service %u opcode %u index %u", cmd->hdr.service, cmd->hdr.opcode,
-			cmd->hdr.index);
+		LOG_DBG("cmd service 0x%02X opcode 0x%02X index 0x%02X", cmd->hdr.service,
```
The BTP spec uses lower-case hex, so seems like `0x%02x` would be more appropriate. I believe it's the common form in most of the Bluetooth code base too.

sjanc commented on tests/bluetooth/tester/src/btp.c:
```diff
@@ -96,8 +96,8 @@ static void cmd_handler(void *p1, void *p2, void *p3)
 
-		LOG_DBG("cmd service %u opcode %u index %u", cmd->hdr.service, cmd->hdr.opcode,
-			cmd->hdr.index);
+		LOG_DBG("cmd service 0x%02x opcode 0x%02x index 0x%02x", cmd->hdr.service,
```
services are defined as decimal

Thalley replied:
:upside_down_face:

Thalley commented:
@sjanc 
In Zephyr the services are defined as decimal, opcodes and error codes are hex, indexes I'm not sure

In the autopts client everything seems to be logged as decimal, e.g. `Header(svc_id=0, op=128, ctrl_index=255, data_len=0)`, but where services are (luckily) also decimal and opcodes are hex. 

We really should align these so that it's possible to compare not only the definitions, but also the logs. 

Is there a reason why the services are decimal?

sjanc reviewed (dismissed):
no reason (other than it is defined like this in autopts docs),  we should just switch to hex everywhere

Thalley committed:
tests: Bluetooth: Tester: Redefine service IDs as hex

Everything else is defined as hex, so it makes sense to
be consistent. This will also make it easier to find the
service IDs in the logs that primarily already log
commands and events as hex.

Signed-off-by: Emil Gydesen <emil.gydesen@nordicsemi.no>

Thalley commented:
Since https://github.com/auto-pts/auto-pts/pull/1292 got merged in the auto-pts repo, I pulled in the commit to also change the service to be hexidecimal in this PR.
